### PR TITLE
Set up Hamburger menu for mobile version

### PR DESCRIPTION
**Pull request by chelobotix**
- In this PR, I fulfilled the following tasks:
- [x] Create a Hamburguer Menu for the mobile version
- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [x] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.
- [x] Update README file with new features

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +47,20 @@
...
+          <a class="hamburguer-link" href="#Portoflio"><h3>Portfolio</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#About"><h3>About</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#Contact"><h3>Contact</h3></a>
+        </li>
+      </ul>
+    </div>
```
Comment (anagudelogu) [inline index.html]: - [x] You did a great job working on your mobile menu! 💯 However, there's still some room to improve the design. One of the main goals of this project is that you show your ability to follow as closely as possible a given design, so please try to stick as much as possible to the Figma template! 🎨 Please consider increasing the font-size and the height of the menu to fill the entire screen. 🤓 🛠️ 


|Template|Your design|
|---|---|
| image | image |

File: hamburguer.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+btnClose.addEventListener('click', () => {
+  divHamburguer.classList.add('run-out');
+  divHamburguer.classList.remove('run');
+});
+
+iconHamburguer.addEventListener('click', () => {
+  divHamburguer.classList.add('run');
+  divHamburguer.classList.remove('run-out');
+});
```
Comment (anagudelogu) [inline hamburguer.js]: - [x] **[OPTIONAL]:** You did a great job writing JavaScript! 💯 However, at the moment, users can scroll over the page when the menu is open, which isn't a great user experience. If you would like to improve further this menu, I suggest you prevent the body from scrolling when it is open. You can achieve this by toggling a class that adds "overflow: hidden" to the body and removes it when hamburger/close buttons are clicked respectively. 🤓 🛠️ 

This is just an idea, feel free to implement your own solution! 🤓 

_--- JS ---_

```js
btnClose.addEventListener('click', () => {
  divHamburguer.classList.add('run-out');
  divHamburguer.classList.remove('run');
  
  body.classList.remove("no-scroll")
});

iconHamburguer.addEventListener('click', () => {
  divHamburguer.classList.add('run');
  divHamburguer.classList.remove('run-out');
  
  body.classList.add("no-scroll")
});
```

_-- CSS ---_

```CSS
.no-scroll {
  overflow: hidden;
}
```

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi Team! @chelobotix @adelphinsucasa, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chelobotix** [comment]
All the observations mentioned in the above comments have been implemented.
Thank you

**Comment by chelobotix** [comment]
@anagudelogu dear friend, you just approved my project on the Microverse dashboard. Anyway, I implemented all the changes that you requested.
Have a nice day

**Comment by anagudelogu** [comment]
> @anagudelogu dear friend, you just approved my project on the Microverse dashboard. Anyway, I implemented all the changes that you requested. Have a nice day

Hi @chelobotix! thanks for letting me know, I'm starting the process to change the status, please stay active on slack, once resolved I'm going to notify you so you can submit your project again. Cheers and happy coding! 🍾

**Comment by enniopo** [comment]
Hi @chelobotix, you should be able to ask for another code review request now. Please let me know if you have any problems, and apologies for the inconvenience.

**Comment by chelobotix** [comment]
Dear @enniopo @anagudelogu,  I've requested a new review and it was approved directly without a new code review intervention. May I merge it now?

**Comment by enniopo** [comment]
Hi @chelobotix - apparently something went wrong in the background. 
I just reset your dashboard again, and I requested a code review request on your behalf so one of our CR can review your project. You should get receive some feedback shortly.

**Review by mikethreels — APPROVED** [review]
Hi Marcelo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions made by the previous CR.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**